.ci/check-commit.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+
+set -e
+
+SHELL_FOLDER=$(
+    cd $(dirname $0)
+    pwd
+)
+
+# check_script=gofmt -e -s -w 
+check_script="goimports -w"
+commit_limit=2
+file_limit=35
+insert_limit=800
+delete_limit=500
+new_file_header_length=35
+skip_check_words="sync code"
+
+LOG_ERROR() {
+    content=${1}
+    echo -e "\033[31m${content}\033[0m"
+}
+
+LOG_INFO() {
+    content=${1}
+    echo -e "\033[32m${content}\033[0m"
+}
+
+execute_cmd() {
+    command="${1}"
+    eval ${command}
+    ret=$?
+    if [ $ret -ne 0 ]; then
+        LOG_ERROR "FAILED of command: ${command}"
+        exit 1
+    else
+        LOG_INFO "SUCCESS of command: ${command}"
+    fi
+}
+
+function check_codeFormat() {
+    # Redirect output to stderr.
+    exec 1>&2
+    go get golang.org/x/tools/cmd/goimports
+    sum=0
+    for file in $(git diff-index --name-status HEAD^ -- | grep -v D | grep -E '\.go' | awk '{print $2}'); do
+        execute_cmd "$check_script $file"
+        sum=$(expr ${sum} + $?)
+    done
+
+    if [ ${sum} -ne 0 ]; then
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/FISCO-BCOS/go-sdk/pullRequest?prid=5569463)